### PR TITLE
Django upgrade

### PR DESCRIPTION
**Pull request by webbyfox**
This changeset updates the Django version of the app to 2.2.19

 - [x] Changelog entry added.
 - [x] Upgraded any vulnerable dependencies.
 - [x] Requirements have been compiled.

### CODE REVIEW

File: app/navigator/urls.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
-from django.conf.urls import url, include
+from django.conf.urls import url
+from django.urls import include, path
```
Comment (stevejalim) [inline app/navigator/urls.py:2]: Do we need `path` here?

File: app/navigator/urls.py [hunk 2]
```diff
@@ -12,7 +13,7 @@
     url(r'^robots\.txt$', TemplateView.as_view(template_name='robots.txt', content_type='text/plain'), name='robots'),
     url(r'^ping\.json$', PingView.as_view(), name='ping'),
     url(r'^grappelli/', include('grappelli.urls')),
-    url('^auth/', include('authbroker_client.urls', namespace='authbroker', app_name='authbroker_client')),
```
Comment (stevejalim) [inline app/navigator/urls.py:15]: Am assuming that authbroker_client defines an app_name in its apps.py - have you checked this?

File: requirements.in [hunk 3]
```diff
@@ -2,7 +2,7 @@ appdirs==1.4.3
 boto3==1.7.31
 contextlib2==0.5.4
 dj-database-url==0.4.1
-Django==1.11.29
+Django>=1.11
```
Comment (stevejalim) [inline requirements.in:5]: Do we want to cap this at 2.x?

**Review by stevejalim — APPROVED** [review]
Looks OK to me with some optional comments - though limiting the Django version is probably worth doing

File: requirements.in (hunk 3 above)
Comment (webbyfox) [reply requirements.in:5]: Good catch

**Pull request merged**